xmeffects.js

```diff
@@ -1,8 +1,14 @@
+(function (window) {
+if (!window.XMPlayer) {
+  window.XMPlayer = {};
+}
+var player = window.XMPlayer;
```

Comment:
`var player = window.XMPlayer` is just by reference. Any changes to `window.XMPlayer` will reflect on `player`, I just didn't want to write the longer name everywhere :)